packages/ember/tests/error_handler_test.js

```diff
@@ -20,7 +22,7 @@ QUnit.module('error_handler', {
     Ember.testing = TESTING;
     window.onerror = WINDOW_ONERROR;
     if (Ember.Test) {
-      Ember.Test.adapter = ADAPTER;
+      TestAdapter = ADAPTER;
```

Comment:
I believe imported variables are treated as const and you can't assign to them like this.